2025-08-20 14:46:36.402 +03:00 [ERR] An error has occurred traceId: 0HNEVRSMKJF2G:00000003
System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 28
2025-08-20 15:38:19.873 +03:00 [ERR] An error has occurred traceId: 0HNEVSPJNCR4E:00000003
System.Collections.Generic.KeyNotFoundException: item with id of 20 is not found
   at TaskTrackerClean.Infrastructure.Repositories.GenericRepository`1.FindByIdAsync(Int32 id) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.Infrastructure\Repositories\GenericRepository.cs:line 56
   at TaskTrackerClean.Application.Services.TaskService.FindByIdAsync(Int32 id) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.Application\Services\TaskService.cs:line 65
   at TaskTrackerClean.API.Controllers.TasksController.GetById(Int32 id) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.API\Controllers\TasksController.cs:line 41
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 28
2025-08-20 15:38:40.423 +03:00 [ERR] An error has occurred traceId: 0HNEVSPJNCR4E:00000004
System.ArgumentException: Unknown priority type has been entered
   at TaskTrackerClean.Application.Helpers.EnumHelper.GetPriority(String priorityText) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.Application\Helpers\EnumHelper.cs:line 36
   at TaskTrackerClean.Application.Services.TaskService.FindAsync(FindTaskDto dto, Int32 page, Int32 pageSize) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.Application\Services\TaskService.cs:line 71
   at TaskTrackerClean.API.Controllers.TasksController.Find(FindTaskDto dto, Nullable`1 page, Nullable`1 pageSize) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.API\Controllers\TasksController.cs:line 48
   at lambda_method28(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 28
